packages/shared/src/HeaderButton.js

```diff
@@ -72,4 +76,14 @@ const styles = StyleSheet.create({
       margin: 16,
     },
   },
+  innerLeftContainer: {
+    ios: {
+      marginStart: 10,
```

Comment:
This is too much duplication, seems shorter to have a specific style. And then:

```js
style={[styles.innerContainer, styles.right]}
```

Where `right` contain the specifics overrides.